activerecord/test/cases/fixtures_test.rb

```diff
@@ -1589,3 +1591,18 @@ def readonly_config
       end
   end
 end
+
+class StrictLoadingFixturesTest < ActiveRecord::TestCase
+  fixtures :strict_zines
+
+  test "strict loading violations are ignored on fixtures" do
+    ActiveRecord::FixtureSet.reset_cache
+    create_fixtures("strict_zines")
+
+    assert_raises(ActiveRecord::StrictLoadingViolationError) do
```

Comment:
this assertion is just to show that strict loading is enabled on this model. by contrast, the next line doesn't crash.